calendar_reboot.js

```diff
@@ -243,15 +246,13 @@ $(document).ready(function() {
         if (confirmation === true) {
             
             var currentCalendarId = loadFromLocalStorage(current_active_calendar);
-            console.log(currentCalendarId);
-            delete calendarUniqueId[currentCalendarId];
-            storeInLocalStorage(masterKey, calendarUniqueId);
-            
             removeFromCalendarDropdown(currentCalendarId);
-            //remove the calendarstate from localStorage
-            removeFromLocalStorage(currentCalendarId);
             
-            removeFromLocalStorage(current_active_calendar);
+            
+            //below does not work
```

Comment:
I explained above why it doesn't work. I don't have access to the calendar object. I'd have to create a new one......